bootstrap.sh

```diff
@@ -10,7 +10,7 @@ CURRENT_DIR=$(pwd)
 # default to clang
 CXX=${CXX:-clang++}
 TARGET=${TARGET:-Release}
-OSRM_RELEASE=${OSRM_RELEASE:-"develop"}
+OSRM_RELEASE=${OSRM_RELEASE:-"rewrite/new-api"}
```

Comment:
:grin: yes @freenerd as soon as I get this damn travis segfault sorted I will pull that in